### PR TITLE
UI-tests

### PR DESCRIPTION
**Pull request by gillianpierce**
Harrow, 

Just wanted to do a preliminary UI test review to make sure there were no glaringly awful things I was doing before making more tests. If you chums could look it over and give me an "all clear" or a "HOLD UP" that would be swell 🤠

Thanks a bunch, 

Gilligan

P.S. I made some changes to our friendly gradle file so pls be mindful of that!!!!

### CODE REVIEW

File: app/src/androidTest/java/com/stl/skipthelibrary/US010101Test.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+@RunWith(AndroidJUnit4.class)
+public class US010101Test extends ActivityTestRule<MyBooksActivity>{
+
+    private Solo solo;
+
+    public US010101Test() {
+        super(MyBooksActivity.class, false, true);
```
Comment (LukeSlev) [inline app/src/androidTest/java/com/stl/skipthelibrary/US010101Test.java]: @gillianpierce There a reason you have touch mode off here? I just ask cause pretty sure we had it on in the lab

Comment (gillianpierce) [reply app/src/androidTest/java/com/stl/skipthelibrary/US010101Test.java]: Nah dawg I copied that part exactly from the finalized lab code github repo

Comment (LukeSlev) [reply app/src/androidTest/java/com/stl/skipthelibrary/US010101Test.java]: buddy did it different in lab i swear

Comment (LukeSlev) [reply app/src/androidTest/java/com/stl/skipthelibrary/US010101Test.java]: whatever this is probably good, just keep on going @gillianpierce

**Pull request merged**